src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <a href="http://localhost:3001/">Home</a>
```

Comment:
So what you want to do here is make this anchor tags Link elements. Like so:
```suggestion
          <Link to="/">Home</Link>
```

This is how you actually use react router.